logger/migrations/0000_init.sql

```diff
@@ -1,7 +1,6 @@
 CREATE TABLE IF NOT EXISTS logs (
     deployment_id TEXT,        -- The deployment that this log line pertains to.
     shuttle_service_name TEXT, -- The shuttle service which created this log.
-    timestamp INTEGER,         -- Unix epoch timestamp.
-    level INTEGER,             -- The log level
-    fields TEXT                -- Log fields object.
+    tx_timestamp INTEGER,      -- Unix epoch timestamp.
```

Comment:
This is millisecond accuracy right? Feels weird to go from nanosecond timestamps to millis. Micros could be of interest. (what if logs get out of order due to this?)

Reply:
We'll have though to see how the proto timestamp gets serialized to see if it can fit `INTEGER` type here, which can be at maximum 8 bytes, but this is also signed. This is a good catch!